webextensions/api/extension.json

```
@@ -384,7 +384,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/extension/sendRequest",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "4"
```

:x: Like `onRequest`.

```suggestion
                "version_added": "5"
```